main.cpp

```diff
@@ -46,33 +107,49 @@ struct BoulderProblem
     {
         int holdType = 2;
         double holdSize = 0.8; 
+        double holdHeight;
     };
 
-    void calculateDifficulty( int problemGrade, Hold hold );
+    double calculateDifficulty( double ropeLength );
 
     Hold crimp;
 };
 
+double BoulderProblem::calculateDifficulty( double ropeLength )
+{
+    double difficulty = ( ( problemGrade * crimp.holdType ) / crimp.holdSize ) - ropeLength;
+    return difficulty;
+}
```

Comment:
- [ ] remove local variable.
- [ ] remove all those spaces around the `(` and `)`